src/AnalyzerConfiguration/NI.ruleset

```diff
@@ -399,46 +386,6 @@
     <Rule Id="SA1652" Action="None" />
     <Rule Id="SX1309" Action="None" />
   </Rules>
-  <Rules AnalyzerId="System.Collections.Immutable.Analyzers" RuleNamespace="System.Collections.Immutable.Analyzers">
-    <Rule Id="RS0012" Action="None" />
-  </Rules>
-  <Rules AnalyzerId="System.Resources.CSharp.Analyzers" RuleNamespace="System.Resources.CSharp.Analyzers">
-    <Rule Id="CA1824" Action="Warning" />
-  </Rules>
-  <Rules AnalyzerId="System.Runtime.Analyzers" RuleNamespace="System.Runtime.Analyzers">
-    <Rule Id="CA1304" Action="Warning" />
-    <Rule Id="CA1305" Action="Warning" />
-    <Rule Id="CA1307" Action="None" />
-    <Rule Id="CA1308" Action="Warning" />
-    <Rule Id="CA1813" Action="Warning" />
-    <Rule Id="CA1816" Action="Warning" />
-    <Rule Id="CA1820" Action="Warning" />
-    <Rule Id="CA2002" Action="Warning" />
-    <Rule Id="CA2208" Action="Warning" />
-    <Rule Id="CA2216" Action="None" />
-    <Rule Id="CA2241" Action="None" />
-    <Rule Id="CA2242" Action="Warning" />
-    <Rule Id="CA2243" Action="None" />
-    <Rule Id="RS0014" Action="None" />
-  </Rules>
-  <Rules AnalyzerId="System.Runtime.CSharp.Analyzers" RuleNamespace="System.Runtime.CSharp.Analyzers">
-    <Rule Id="CA1309" Action="None" />
-    <Rule Id="CA1601" Action="Warning" />
-    <Rule Id="CA1810" Action="Warning" />
-    <Rule Id="CA1825" Action="None" />
-    <Rule Id="CA2201" Action="Warning" />
-    <Rule Id="CA2207" Action="Warning" />
-    <Rule Id="CA2215" Action="Warning" />
-  </Rules>
-  <Rules AnalyzerId="System.Runtime.InteropServices.CSharp.Analyzers" RuleNamespace="System.Runtime.InteropServices.CSharp.Analyzers">
-    <Rule Id="RS0015" Action="None" />
-  </Rules>
-  <Rules AnalyzerId="System.Threading.Tasks.Analyzers" RuleNamespace="System.Threading.Tasks.Analyzers">
-    <Rule Id="RS0018" Action="None" />
-  </Rules>
-  <Rules AnalyzerId="Text.CSharp.Analyzers" RuleNamespace="Text.CSharp.Analyzers">
-    <Rule Id="CA1704" Action="Warning" />
-  </Rules>
   <Rules AnalyzerId="Roslynator.CSharp.Analyzers" RuleNamespace="Roslynator.CSharp.Analyzers">
```

Comment:
Re-saving the ruleset puts these analyzers above StyleCop as these start with "R". Once this change and the other are made, editing this file with the official ruleset editor seems possible (it only removes comments).

Reply:
It's hard to understand what you're calling out specifically since we can't highlight sections of code, but I assume you mean SA vs. RS codes, so I will move RS above SA. If that isn't what you were talking about, please clarify

Reply:
nevermind, I think you're saying Roslynator vs StyleCop in the AnalyzerId ... I'll sort them all